src/libs/logger/creporterlogger.cpp

```diff
@@ -121,18 +124,25 @@ void CReporterLogger::messageHandler(QtMsgType type,
                                      const QMessageLogContext &context,
                                      const QString &msg)
 {
-    Q_UNUSED(context);
+    Q_ASSERT(CReporterLogger::sm_LogType != CReporter::LogNone);
 
-    if (CReporterLogger::sm_LogType == CReporter::LogNone) {
+    QString logMessage = qFormatLogMessage(type, context, msg);
+
+    // print nothing if message pattern didn't apply / was empty.
+    // (still print empty lines, e.g. because message itself was empty)
+    if (logMessage.isNull())
```

Comment:
yup, as the comment says: "still print empty lines, e.g. because message itself was empty"